### PR TITLE
[JENKINS-58716] Using gitHubRepo

### PR DESCRIPTION
**Pull request by jglick**
Matches https://github.com/jenkinsci/incrementals-tools/pull/16.

### CODE REVIEW

File: empty-plugin/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
         <changelist>-SNAPSHOT</changelist>
         <jenkins.version>2.235.1</jenkins.version>
         <java.level>8</java.level>
+        #if( $hostOnJenkinsGitHub == "true" )<gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>#end
     </properties>
     <name>TODO Plugin</name>
     #if( $hostOnJenkinsGitHub == "true" )<url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>#end
```
Comment (jglick) [inline empty-plugin/src/main/resources/archetype-resources/pom.xml:22]: Deliberately _not_ touching `/project/url`.

File: common-files/.mvn/extensions.xml [hunk 2]
```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.1</version>
+    <version>1.2-SNAPSHOT</version> <!-- TODO https://github.com/jenkinsci/incrementals-tools/pull/16 -->
```
Comment (jglick) [inline common-files/.mvn/extensions.xml:5]: Cannot prepare a timestamped snapshot, so waiting for upstream to be approved to merge and release.

**Review by timja — APPROVED** [review]

**Comment by jglick** [comment]
Probably ready to go but I would like to first get a deployment from https://github.com/jenkinsci/workflow-step-api-plugin/pull/58 to verify, and apparently Incrementals deployment is currently broken.

File: empty-plugin/src/main/resources/archetype-resources/pom.xml (hunk 1 above)
Comment (daniel-beck) [reply empty-plugin/src/main/resources/archetype-resources/pom.xml:22]: Could you explain why?

**Pull request merged**